pinot-common/src/main/java/org/apache/pinot/common/function/AggregationFunctionType.java

```diff
@@ -75,15 +75,6 @@ public String getName() {
     return _name;
   }
 
-  public boolean isOfType(AggregationFunctionType... aggregationFunctionTypes) {
-    for (AggregationFunctionType aggregationFunctionType : aggregationFunctionTypes) {
-      if (this == aggregationFunctionType) {
-        return true;
-      }
-    }
-    return false;
-  }
```

Comment:
This appears to be untested, but also unused. Is there a reason to keep this?

Reply:
We can remove it